QuadradiusR/prefabs/torus.tscn

```diff
@@ -230,6 +216,54 @@ tracks/3/keys = {
 "values": [ Vector2( 1, 1 ) ]
 }
 
+[sub_resource type="Animation" id=10]
```

Comment:
This and below handles selecting owned piece to open the powerup list (right click), below handles tinting the piece according to its color to show it has powerups